.github/workflows/python-package.yml

```diff
@@ -35,13 +35,25 @@ jobs:
         # extend-select will add rules in addition to defaults
         flake8 . --extend-select=E9,F63,F7,F82,W605 --count --exit-zero --max-complexity=10 --statistics
   build:
-    runs-on: ubuntu-latest
     needs: formatting
+    runs-on: ${{ matrix.os }}
+    name: Build ${{ matrix.os }} - ${{ matrix.arch }} - Python ${{ matrix.python-version }} - ${{ matrix.lxml-dep_name }}
```

Comment:
Looks like this didn't come through. I think you can just do `matrix.lxml-dep` and it will show as empty string for non-lxml, which I think is okay?

Reply:
Looks good now, thank you!